src/xnnpack/internal.h

```diff
@@ -103,7 +103,7 @@ enum xnn_status xnn_setup_batch_matrix_multiply_nc_qp8_f32_qc8w(
     xnn_operator_t batch_matrix_multiply_op,  //
     void* workspace,                          //
     const int8_t* input_a,                    //
-    const int8_t* input_b,                    //
+    const float* input_b,                     //
```

Comment:
this doesnt look right.  the input_b is qc8w which is int8_t

Reply:
But the function definition in `src/operators/batch-matrix-multiply-nc.c` uses `const float* input_b`.